app/javascript/styles/mastodon/statuses.scss

```diff
@@ -83,9 +83,14 @@
   background: $ui-highlight-color;
   color: $primary-text-color;
   text-transform: none;
-  line-height: 36px;
+  line-height: 16px;
   height: auto;
-  padding: 3px 15px;
+  min-height: 36px;
+  min-width: 88px;
```

Comment:
I'm not sure what purpose the `min-width` serves? Also I'm not completely sure about reducing the font size in most cases…

Reply:
When reducing the left and right padding, we have added restrictions so that the buttons are not too small and cramped in languages such as Arabic that have a smaller character width than English.

The font-size reduction avoids line breaks in languages that may fit on fewer lines, while comparing the display in each language, and prioritizes the appearance of two or three lines.

But I didn't want to do that in a language like English that fits well without shrinking. If necessary, I think you can add hints by specifying the language. Do you have any ideas?
![min-width-1](https://user-images.githubusercontent.com/28195220/103953942-7c6e4100-5186-11eb-93a9-6e3788f6b1af.jpg)